src/pretix/control/views/global_settings.py

```diff
@@ -83,3 +83,20 @@ class RefundDetailView(AdministratorPermissionRequiredMixin, View):
     def get(self, request, *args, **kwargs):
         p = get_object_or_404(OrderRefund, pk=request.GET.get('pk'))
         return JsonResponse({'data': p.info_data})
+
+
+class AdminiInformationView(AdministratorPermissionRequiredMixin, FormView):
+    template_name = 'pretixcontrol/global_info.html'
+    form_class = AdminInformationForm
+
+    def form_valid(self, form):
+        form.save()
+        messages.success(self.request, _('Your changes have been saved.'))
+        return super().form_valid(form)
+
+    def form_invalid(self, form):
+        messages.error(self.request, _('Your changes have not been saved, see below for errors.'))
+        return super().form_invalid(form)
+
+    def get_success_url(self):
+        return reverse('control:admin.information')
```

Comment:
Please configure your editor to automatically add one blank line at the end of file.

Reply:
sure,configured the Editor.